demo/index.tsx

```diff
@@ -1,55 +1,60 @@
-import { render } from 'react-dom';
-import * as React from 'react';
-
+import { render } from "react-dom";
+import * as React from "react";
 import {
-  MuiThemeProvider, createMuiTheme,
-  Avatar, Icon,
-} from '@material-ui/core';
-import { blue, pink } from '@material-ui/core/colors';
+  MuiThemeProvider,
+  createMuiTheme,
+  Avatar,
+  Icon,
+  Divider,
+} from "@material-ui/core";
+import { blue, pink } from "@material-ui/core/colors";
 
-import { Timeline, IEvent } from '../src/timeline/timeline';
+import { Timeline, IEvent } from "../src/timeline/timeline";
 
 const theme = createMuiTheme({
   palette: {
     primary: blue,
-    secondary: pink
-  }
+    secondary: pink,
+  },
 });
 
-const icon =
+const icon = (
   <Avatar>
     <Icon>work</Icon>
-  </Avatar>;
+  </Avatar>
+);
 
 const events: IEvent[] = [
   {
-    time: new Date(),
-    title: 'Event 1',
-    description: [ 'Some description for event 1' ],
-    icon
+    title: "Event 1",
+    description: ["Some description for event 1"],
+    icon,
   },
   {
-    time: new Date(),
-    title: 'Event 2',
-    description: [ 'Some description for event 2' ],
-    icon
+    title: "Event 2",
+    description: ["Some description for event 2"],
+    icon,
   },
   {
-    time: new Date(),
-    title: 'Event 3',
-    description: [ 'Some description for event 3' ],
-    icon
-  }
+    title: "Event 3",
+    description: ["Some description for event 3"],
+    icon,
+  },
 ];
 
 class Demo extends React.Component {
   render() {
     return (
       <MuiThemeProvider theme={theme}>
-	<Timeline events={events}/>
+        <Timeline events={events} />
+        <Divider style={{ marginTop: 64, marginBottom: 64 }} />
+        <Timeline
+          events={events}
+          styles={{ rightCard: { textAlign: "right" } }}
```

Comment:
`rightCard` is a CSS class name?

Reply:
Not exactly. Mind have a look at `Timeline` properties: 
```ts
export interface IProps {
  events: IEvent[];
  styles?: IStyles;
}
```
So `styles` is a component property. Now, the definition of `styles`:
```ts
export interface IStyles {
  leftCard?: React.CSSProperties;
  rightCard?: React.CSSProperties;
}
```
So `styles` can contain CSS styles separately for left and right cards. 

In essence, `styles` contains a map in which key is a timeline element and value is a map of CSS styles to apply to the corresponding element.

Your comment makes me wonder if naming that property `styles` is a good idea. Let me know if you think we should figure out the better name!